src/api/api_arith.cpp

```diff
@@ -121,7 +121,7 @@ extern "C" {
 
     Z3_bool Z3_API Z3_is_algebraic_number(Z3_context c, Z3_ast a) {
         LOG_Z3_is_algebraic_number(c, a);
-        return mk_c(c)->autil().is_irrational_algebraic_numeral(to_expr(a)) ? Z3_TRUE : Z3_FALSE;
+        return mk_c(c)->autil().is_irrational_algebraic_numeral(to_expr(a)) ? true : false;
```

Comment:
if we are extra courageous, we may even do

      return mk_c(c)->autil().is_irrational_algebraic_numeral(to_expr(a))